src/components/modular-content/rich-text/rich-text.scss

```diff
@@ -0,0 +1,30 @@
+.rich-text {
+  a {
+    text-decoration: none;
+    color: clr(link);
+
+    &:focus,
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  p {
+    margin-bottom: 26px;
```

Comment:
set in relative `em`

Reply:
and margin for all descendants
```css
  > * + * {
    margin-top: 1em;
  }
```

Reply:
pls do this when implementing real styles, this was just copied from other project #mvp